### DISCUSSION TITLE
Protocol analysers

### DISCUSSION THREAD

**Discussion by AFUDIrk** [gusmanb/logicanalyzer#89] [Q&A]
Hello can someone help me?
I cannot select a protocol under Protocol analyzers, even though the three standard protocols exist in the Analyzers directory.

**Comment by AFUDIrk** [comment]
![Screenshot](https://github.com/gusmanb/logicanalyzer/assets/11766417/34af1bb3-3ae3-4ce1-9a02-126be5da9b19)

**Reply by gusmanb** [reply]
Do a capture, until there is some data to analyze the menu stays disabled.

**Reply by cdwijs** [reply]
> Do a capture, until there is some data to analyze the menu stays disabled.

What is the technical reason behind that decision? I would like to prepare my decoders before I start the capture, so I see the decoding of the signal as it comes in.

**Comment by AFUDIrk** [comment]
Thank you for the info.  I haven't received all the parts yet and will probably finish the device this weekend.  then I will continue testing.  I was just surprised because this field wasn't grayed out in the description.

**Comment by gusmanb** [comment]
If you want to try it, here is an I2C capture, you can open it from the file menu :)

[I2C 1Mhz.zip](https://github.com/gusmanb/logicanalyzer/files/13128251/I2C.1Mhz.zip)

**Comment by AFUDIrk** [comment]
Thank you very much, I'll take a look tomorrow.  The pin and socket headers
arrived today.  Now the IC's from the level shifter are still missing.

Mit freundlichen Grüßen
Dirk König

Agustín Gimenez Bernad ***@***.***> schrieb am Mi., 25. Okt.
2023, 00:40:
